### PR TITLE
adding loki to the quickstart

### PR DESCRIPTION
**Pull request by joshuafernandes**
Signed-off-by: Joshua Fernandes <joshua.fernandes@consensys.net>

## Pull request checklist

Use the following list to make sure your PR fits the Besu documentation quality standard.

### Before creating the pull request

Make sure that:

- [x] [All commits in this PR are signed off for the DCO](https://wiki.hyperledger.org/display/BESU/DCO).
- [x] You've read the [contribution guidelines](https://wiki.hyperledger.org/display/BESU/Documentation).
- [x] You've [previewed your changes locally](https://wiki.hyperledger.org/display/BESU/Preview+the+documentation).

## Describe the change

Adding documentation for Grafana Loki which has been added to the Dev Quickstart

## Impacted parts

For content changes:

- [x] Documentation content
- [ ] Documentation page organization

For tool changes:

- [ ] CircleCI workflow
- [ ] Build and QA tools (for example, lint or vale)
- [ ] MkDocs templates
- [ ] MkDocs configuration
- [ ] Python dependencies
- [ ] Node dependencies and JavaScript
- [ ] Read the Docs configuration
- [ ] GitHub integration

## After creating your PR and tests have finished

Make sure that:

- [ ] You've fixed any issues raised by the tests.
- [ ] You've [previewed your changes on Read the Docs](https://wiki.hyperledger.org/display/BESU/Preview+the+documentation)
  and added a [preview link](#preview).

### CODE REVIEW

File: docs/private-networks/how-to/monitor/loki.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+description: Using Grafana Loki with Hyperledger Besu
```
Comment (NicolasMassart) [inline docs/private-networks/how-to/monitor/loki.md:2]: For SEO:
```suggestion
description: Using Grafana Loki log management platform with Hyperledger Besu
```

File: docs/private-networks/tutorials/quickstart.md [hunk 2]
```diff
@@ -133,14 +136,19 @@ You can directly access these tools from your browser at the addresses displayed
...
-[performances monitoring documentation](../../public-networks/how-to/monitor/metrics.md),
+[performance monitoring documentation](../../public-networks/how-to/monitor/metrics.md),
 [Prometheus documentation](https://prometheus.io/docs/introduction/overview/)
 and [Grafana documentation](https://grafana.com/docs/).
 
 ![Grafana](../../images/grafana.png)
 
+and collated logs via Grafana Loki
+
+![Loki](../../images/grafana_loki.png)
```
Comment (NicolasMassart) [inline docs/private-networks/tutorials/quickstart.md:150]: For SEO:
```suggestion
![Grafana Loki dashboard screenshot](../../images/grafana_loki.png)
```

File: docs/private-networks/tutorials/quickstart.md [hunk 3]
```diff
@@ -133,14 +136,19 @@ You can directly access these tools from your browser at the addresses displayed
...
 - [Grafana dashboard](http://localhost:3000/d/XE4V0WGZz/besu-overview?orgId=1&refresh=10s&from=now-30m&to=now&var-system=All)
+- [Grafana Loki logs dashboard](http://localhost:3000/d/Ak6eXLsPxFemKYKEXfcH/quorum-logs-loki?orgId=1&var-app=quorum&var-search=)
 
 For more details on how to configure and use these tools for your own nodes, see the
-[performances monitoring documentation](../../public-networks/how-to/monitor/metrics.md),
+[performance monitoring documentation](../../public-networks/how-to/monitor/metrics.md),
 [Prometheus documentation](https://prometheus.io/docs/introduction/overview/)
 and [Grafana documentation](https://grafana.com/docs/).
 
 ![Grafana](../../images/grafana.png)
```
Comment (NicolasMassart) [inline docs/private-networks/tutorials/quickstart.md:146]: For SEO:
```suggestion
![Grafana dashboard screenshot](../../images/grafana.png)
```

**Review by NicolasMassart — COMMENTED** [review]
made a few syntax fix and SEO suggestions (that you can batch merge directly from this PR on Github website) but otherwise it looks good for me.

File: docs/private-networks/how-to/monitor/loki.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+[Grafana Loki] is an open-source log management platform that is available when using the
+[Developer Quickstart](../../tutorials/quickstart.md).
+
+The [Promtail configuration] ingests logs at regular defined intervals and outputs them to [Loki] for storage.
+
+The `pipeline configuration` in Promtail defines pipeline stages that can collate logs natively or using the JSON format.
+
+!!! note
+
+    If using the pipeline regex stage in `Promtail`, configuration must match the your log format.
```
Comment (rolandtyler) [inline docs/private-networks/how-to/monitor/loki.md:16]:
```suggestion
    If using the pipeline regex stage in `Promtail`, configuration must match your log format.
```

**Review by rolandtyler — APPROVED** [review]
Minor suggestion to remove extra word, otherwise LGTM

**Comment by joshuafernandes** [comment]
> made a few syntax fix and SEO suggestions (that you can batch merge directly from this PR on Github website) but otherwise it looks good for me.

Thankyou @NicolasMassart, added em all in :)

File: docs/private-networks/how-to/monitor/loki.md (hunk 4 above)
Comment (joshuafernandes) [reply docs/private-networks/how-to/monitor/loki.md]: Sorted, thankyou @rolandtyler

**Pull request merged**